core/src/main/java/com/linecorp/armeria/common/util/TextFormatter.java

```diff
@@ -121,25 +121,52 @@ public static StringBuilder elapsedAndSize(long startTimeNanos, long endTimeNano
                                           .withZone(ZoneId.of("GMT"));
 
     /**
-     * Formats the given epoch time to typical human-readable format "yyyy-MM-dd'T'HH_mm:ss.SSSX" and appends
-     * it to the specified {@link StringBuilder}.
+     * Formats the given epoch time in milliseconds to typical human-readable format
+     * "yyyy-MM-dd'T'HH_mm:ss.SSSZ" and appends it to the specified {@link StringBuilder}.
      */
     public static void appendEpoch(StringBuilder buf, long timeMillis) {
```

Comment:
What do you think about renaming `*Epoch` methods to `*EpochMillis` and adding their deprecated versions?